docs/sources/alerting/unified-alerting/message-templating/_index.md

```diff
@@ -84,14 +113,20 @@ Template to render a single alert:
 Template to render entire notification message:
 
 ```
-{{ define "message" }}
+{{ define "mymessage" }}
   {{ if gt (len .Alerts.Firing) 0 }}
     {{ len .Alerts.Firing }} firing:
-    {{ range .Alerts.Firing }} {{ template "alert" .}} {{ end }}
+    {{ range .Alerts.Firing }} {{ template "myalert" .}} {{ end }}
   {{ end }}
   {{ if gt (len .Alerts.Resolved) 0 }}
     {{ len .Alerts.Resolved }} resolved:
-    {{ range .Alerts.Resolved }} {{ template "alert" .}} {{ end }}
+    {{ range .Alerts.Resolved }} {{ template "myalert" .}} {{ end }}
   {{ end }}
 {{ end }}
 ```
+
+### HTML in Message Templates
+
+HTML in Alerting message templates is escaped, and rendering of HTML in the resulting notification is not supported.
```

Comment:
```suggestion
Although HTML in alerting message templates is escaped, we do not support rendering of HTML in the resulting notification.
```

Reply:
I considered dropping the "Although", what are your thoughts on this? The fact that HTML is escaped is why we don't support rendering of HTML, it isn't really an alternative.